pkg/cmd/version/version.go

```diff
@@ -84,6 +91,9 @@ func Command(p cli.Params) *cobra.Command {
 		},
 	}
 
+	cmd.Flags().StringVarP(&namespace, "namespace", "n", namespace,
```

Comment:
Just thought instead of this, we should set the global flags here so that we will get all along with namespace

Reply:
I think this will not make much sense here as the need to accept param from ldflag